servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/JDBCCertRecordStoreConnection.java

```diff
@@ -60,7 +65,7 @@
                 "GROUP BY hostName, provider, service" +
             ") AS m on (m.hostName=a.hostName AND m.provider=a.provider AND m.service=a.service AND a.currentTime=date_updated) " +
             "SET lastNotifiedTime=?, lastNotifiedServer=? " +
-            "WHERE a.currentTime < (CURRENT_DATE - INTERVAL 3 DAY) AND " +
+            "WHERE a.currentTime < (CURRENT_DATE - INTERVAL ? DAY) AND " +
```

Comment:
The setting is in hours so we need to update DAY parameter.

Reply:
Right, this was lost when I did the merge... Fixing now